### PR TITLE
Localization: update local_pyramid_map

### PR DESCRIPTION
**Pull request by tthhee**

### CODE REVIEW

File: modules/localization/msf/common/util/file_utility.cc [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+#include <boost/filesystem.hpp>
+#undef BOOST_NO_CXX11_SCOPED_ENUMS
+#include <algorithm>
+#include <iostream>
+#include "fastrtps/TopicDataType.h"
+
+namespace apollo {
+namespace localization {
+namespace msf {
+const size_t BUFFER_SIZE = 20480000;
```
Comment (jinghaomiao) [inline modules/localization/msf/common/util/file_utility.cc]: use kBufferSize type of naming convention

File: modules/localization/msf/common/util/file_utility.cc [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        buf_pt = &buf[count - 1];
+      }
+
+      size_t size = fread(buf_pt, sizeof(unsigned char), BUFFER_SIZE, file);
+      total_size += size;
+
+      ++count;
+    }
+  } else {
+    std::cerr << "Can't find the file: " << file_path << std::endl;
```
Comment (jinghaomiao) [inline modules/localization/msf/common/util/file_utility.cc]: Use AERROR

File: modules/localization/msf/common/util/file_utility.cc [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
```
Comment (jinghaomiao) [inline modules/localization/msf/common/util/file_utility.cc]: Use AERROR

File: modules/localization/msf/common/util/file_utility.h [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+#include <vector>
+#include "cyber/common/log.h"
+
+namespace apollo {
+namespace localization {
+namespace msf {
+
+class FileUtility {
+ public:
+  static const size_t UCHAR_MD5LENTH = 16;
```
Comment (jinghaomiao) [inline modules/localization/msf/common/util/file_utility.h]: use KConst naming format for const variables.

File: modules/localization/msf/common/util/file_utility.cc (hunk 1 above)
Comment (tthhee) [reply modules/localization/msf/common/util/file_utility.cc]: fixed

File: modules/localization/msf/common/util/file_utility.cc (hunk 2 above)
Comment (tthhee) [reply modules/localization/msf/common/util/file_utility.cc]: fixed

File: modules/localization/msf/common/util/file_utility.cc (hunk 3 above)
Comment (tthhee) [reply modules/localization/msf/common/util/file_utility.cc]: fixed

File: modules/localization/msf/common/util/file_utility.h (hunk 4 above)
Comment (tthhee) [reply modules/localization/msf/common/util/file_utility.h]: fixed

**Pull request merged**